src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/SpringFoxConfiguration.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.config
+
+import org.springframework.beans.BeansException
+import org.springframework.beans.factory.config.BeanPostProcessor
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
+import org.springframework.util.ReflectionUtils
+import org.springframework.web.servlet.mvc.method.RequestMappingInfoHandlerMapping
+import springfox.bean.validators.configuration.BeanValidatorPluginsConfiguration
+import springfox.documentation.builders.PathSelectors
+import springfox.documentation.builders.RequestHandlerSelectors
+import springfox.documentation.spi.DocumentationType
+import springfox.documentation.spring.web.plugins.Docket
+import springfox.documentation.spring.web.plugins.WebFluxRequestHandlerProvider
+import springfox.documentation.spring.web.plugins.WebMvcRequestHandlerProvider
+import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.controllers.RestrictedPatentsController
+import java.lang.reflect.Field
+import java.util.stream.Collectors
+
+@Configuration
+@Import(BeanValidatorPluginsConfiguration::class)
+class SpringFoxConfiguration() {
+
+  @Bean
+  fun api(): Docket? {
```

Comment:
Just checking - does this not have to be removed form the existing config? It was in the adj api

Reply:
So there wasn't any existing config - which meant that RP had a bunch of swagger calls documented which we don't want in our API docs - so I put this in so that we no longer show them